Update ups_install scripts

kutschke opened the pull request:
Make PACKAGE_VERSION and argument to the script install.sh ; previously was hard coded.
Remove reference to non-longer-existing environment variable.
Remove noisy printout.

FNALbuild commented:
Hi @kutschke,
You have proposed changes to files in these packages:
- ups_install

which require these tests: build.

@Mu2e/fnalbuild-users, @Mu2e/write have access to CI actions on [main](/Mu2e/Offline/tree/main).





:hourglass: The following tests have been triggered for 29ebcd99774cd2c90c5aad8c7770af682a4d8baf: build  (Build queue is empty)


[About FNALbuild.](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#GitHub_Pull_Request_Procedures_and_FNALbuild) [Code review on Mu2e/Offline.](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#Code_Review)

FNALbuild commented:
:sunny: The build tests passed at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf.


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged 29ebcd99774cd2c90c5aad8c7770af682a4d8baf at df8da92305d6576cdfef3b92d90b7f2922c4da58 |
| build (prof) | :white_check_mark: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/scons.log). Build time: 21 min 24 sec |
| ceSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/ceSimReco.log)  |
| g4test_03MT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/g4test_03MT.log)  |
| transportOnly | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/transportOnly.log)  |
| POT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/POT.log)  |
| g4study | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/g4study.log)  |
| cosmicSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/cosmicOffSpill.log)  |
| ceSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/ceSteps.log)  |
| ceDigi | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/ceDigi.log)  |
| muDauSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/muDauSteps.log)  |
| ceMix | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/ceMix.log)  |
| rootOverlaps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/g4surfaceCheck.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 0 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/fixme_todo.log) |
| clang-tidy | :white_check_mark: | [0 errors 0 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/clang-tidy.log) |
| whitespace check | :white_check_mark: | [no whitespace errors found](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf after being merged into the base branch at df8da92305d6576cdfef3b92d90b7f2922c4da58.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1385/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.

eflumerf reviewed (approved):
LGTM Did you want to make the directory name changes here as well?

kutschke commented:
Thanks Eric.  If you are on board with the directory name changes, let's make them here.  I will send Mackenzie the tarball with the current-style directory names.  She can install that.  I will finish these changes and tag as d10_17_01 and she can make the Jenkins jobs and install those too.

kutschke commented:
A small change in plans.  Let's merge this one today so that Mackenzie can get started.  When I make the directory name changes she will not need to do maintenance on her end.

kutschke commented:
@FNALbuild run build test

FNALbuild commented:
:hourglass: The following tests have been triggered for 29ebcd99774cd2c90c5aad8c7770af682a4d8baf: build  (Build queue is empty)

FNALbuild commented:
:sunny: The build tests passed at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf.


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged 29ebcd99774cd2c90c5aad8c7770af682a4d8baf at 3904c7fd83efc88dab151439e36cc9c41c4347ca |
| build (prof) | :white_check_mark: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/scons.log). Build time: 12 min 00 sec |
| ceSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/ceSimReco.log)  |
| g4test_03MT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/g4test_03MT.log)  |
| transportOnly | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/transportOnly.log)  |
| POT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/POT.log)  |
| g4study | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/g4study.log)  |
| cosmicSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/cosmicOffSpill.log)  |
| ceSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/ceSteps.log)  |
| ceDigi | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/ceDigi.log)  |
| muDauSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/muDauSteps.log)  |
| ceMix | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/ceMix.log)  |
| rootOverlaps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/g4surfaceCheck.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 0 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/fixme_todo.log) |
| clang-tidy | :white_check_mark: | [0 errors 0 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/clang-tidy.log) |
| whitespace check | :white_check_mark: | [no whitespace errors found](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf after being merged into the base branch at 3904c7fd83efc88dab151439e36cc9c41c4347ca.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1387/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.

kutschke commented:
@eflumerf Eric, can you try approving one more time.  Include a trivial comment in the review. I want to see if this was a transient issue or is repeatable.  If it still fails I will ask someone else to approve so that we can see if it affects others.  We did make changes to the permission policies about a month ago and there has not been a lot of activity since then - so it's possible that you are the first non-admin, non-owner to try to approve since then.

eflumerf reviewed (approved):
Still LGTM ;)
Still grey check mark... 😢

kutschke commented:
@resnegfk Hi Krzysztof.  We have some behaviour on this PR that I don't understand.  Eric has already approved it but it is not recorded as an approval that permits the review to proceed.  I have some guesses about what this might be.  To help narrow down the problem, I have added you as a reviewer.  Please approve and we will see what happens.

kutschke commented:
@eflumerf As an experiment, I added you to have explicit read access to the repo.  Please try to approve again.  This is motivated by the phrase "granted explicit access" on the page:

 https://github.com/organizations/Mu2e/settings/code_review_limits

Previously you had implicit but not explicit read access via your membership in the organization.

eflumerf reviewed (approved):
test

eflumerf commented:
Probably need to make the review requirements less strict... https://docs.github.com/en/organizations/managing-organization-settings/managing-pull-request-reviews-in-your-organization Anyone in the Mu2e org should be able to approve PRs, I would think

kutschke commented:
> Probably need to make the review requirements less strict... https://docs.github.com/en/organizations/managing-organization-settings/managing-pull-request-reviews-in-your-organization Anyone in the Mu2e org should be able to approve PRs, I would think

Yes.  That's also what I think we need to do.  I am trying to recover the why we did that story - my guess is we did not fully understand what we are doing.  I will try this later this afternoon.

kutschke commented:
@eflumerf I am 99% sure that I just removed the feature that was limiting your ability to approve.  That, by itself, did not fix the problem.  Please reapprove.

eflumerf reviewed (approved):
test2

kutschke commented:
@eflumerf please try one more time.

eflumerf reviewed (approved):
test3

kutschke commented:
@eflumerf Please try again.  This time I clicked the "read access and higher" button for the repo.

eflumerf reviewed (approved):
test4

eflumerf commented:
At this point, I'm wondering if there's a grandfather clause in effect...

kutschke commented:
Still not working.  I am out of ideas.

eflumerf reviewed (approved):
test5

eflumerf reviewed (approved):
test6

eflumerf commented:
Try closing this PR then opening a new one with the same branch? Of course then you'd have to run the build bot again...

kutschke commented:
@brownd1978 I suggest that we proceed as follows.  If you feel comfortable doing so, approve this yourself to remove the block and merge at your convenience. I will continue to debug this problem using PR #837 which was opened just for that purpose.  I will create kutschke-non-admin github identity for myself and make it a Mu2e member so that the others can go back to their day jobs.

brownd1978 commented:
Sounds like a good plan.
This problem must be related to the recent protections we added to the
Offline repo, we have had many PRs approved by non-admin reviewers
previously.
Dave

On Mon, Aug 29, 2022 at 06:44 Rob Kutschke ***@***.***> wrote:

> @brownd1978 <https://github.com/brownd1978> I suggest that we proceed as
> follows. If you feel comfortable doing so, approve this yourself to remove
> the block and merge at your convenience. I will continue to debug this
> problem using PR #837 <https://github.com/Mu2e/Offline/pull/837> which
> was opened just for that purpose. I will create kutschke-non-admin github
> identity for myself and make it a Mu2e member so that the others can go
> back to their day jobs.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Mu2e/Offline/pull/835#issuecomment-1230329404>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAH576EEMTRQTEMKR4TB53V3S5ERANCNFSM57UU4FGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
David Brown            ***@***.***
Office Phone (510) 486-7261
Lawrence Berkeley National Lab
M/S 50R5008 (50-6026C)  Berkeley, CA 94720

kutschke commented:
> This problem must be related to the recent protections we added to the Offline repo, we have had many PRs approved by non-admin reviewers previously.

Yes.  Since we added the protections, every PR has had at least one approver with write or better privs to the repo.

FNALbuild commented:
:memo: The HEAD of `main` has changed to f39753cc2547575e3246e031401eac79db824660. Tests are now out of date.

kutschke commented:
@FNALbuild run build test

FNALbuild commented:
:hourglass: The following tests have been triggered for 29ebcd99774cd2c90c5aad8c7770af682a4d8baf: build  (Build queue is empty)

FNALbuild commented:
:sunny: The build tests passed at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf.


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged 29ebcd99774cd2c90c5aad8c7770af682a4d8baf at 2b95911590ef3a9a1d561d9db09f80761d51114e |
| build (prof) | :white_check_mark: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/scons.log). Build time: 11 min 02 sec |
| ceSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/ceSimReco.log)  |
| g4test_03MT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/g4test_03MT.log)  |
| transportOnly | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/transportOnly.log)  |
| POT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/POT.log)  |
| g4study | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/g4study.log)  |
| cosmicSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/cosmicOffSpill.log)  |
| ceSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/ceSteps.log)  |
| ceDigi | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/ceDigi.log)  |
| muDauSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/muDauSteps.log)  |
| ceMix | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/ceMix.log)  |
| rootOverlaps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/g4surfaceCheck.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 0 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/fixme_todo.log) |
| clang-tidy | :white_check_mark: | [0 errors 0 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/clang-tidy.log) |
| whitespace check | :white_check_mark: | [no whitespace errors found](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf after being merged into the base branch at 2b95911590ef3a9a1d561d9db09f80761d51114e.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1392/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.

kutschke commented:
Ray just pointed out that this needs additional work before it is useful.  So I changed this PR to draft status and am working on it this afternoon.

Last night when I removed "require approvals" from the branch protection rules, I missed that it wanted my GitHub password.  So the change only took effect a few minutes ago when I did it correctly.

FNALbuild commented:
:memo: The HEAD of `main` has changed to d470df9baf5f24f141830c64ea4a235dd54b98bc. Tests are now out of date.

kutschke commented:
I will add the other features as a separate PR.  I just learned that this PR is needed soon.

kutschke commented:
@FNALbuild run build test

FNALbuild commented:
:hourglass: The following tests have been triggered for 29ebcd99774cd2c90c5aad8c7770af682a4d8baf: build  (Build queue has 1 jobs)

FNALbuild commented:
:sunny: The build tests passed at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf.


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| test with | :white_check_mark: | Command did not list any other PRs to include |
| merge | :white_check_mark: | Merged 29ebcd99774cd2c90c5aad8c7770af682a4d8baf at 33c93366aadd5b2f7eb2a00561042e97cc31b710 |
| build (prof) | :white_check_mark: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/scons.log). Build time: 11 min 13 sec |
| ceSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/ceSimReco.log)  |
| g4test_03MT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/g4test_03MT.log)  |
| transportOnly | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/transportOnly.log)  |
| POT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/POT.log)  |
| g4study | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/g4study.log)  |
| cosmicSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/cosmicOffSpill.log)  |
| ceSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/ceSteps.log)  |
| ceDigi | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/ceDigi.log)  |
| muDauSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/muDauSteps.log)  |
| ceMix | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/ceMix.log)  |
| rootOverlaps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/g4surfaceCheck.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 0 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/fixme_todo.log) |
| clang-tidy | :white_check_mark: | [0 errors 0 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/clang-tidy.log) |
| whitespace check | :white_check_mark: | [no whitespace errors found](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/artifact/whitespace_errs.log) |


N.B. These results were obtained from a build of this Pull Request at 29ebcd99774cd2c90c5aad8c7770af682a4d8baf after being merged into the base branch at 33c93366aadd5b2f7eb2a00561042e97cc31b710.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-offline-build-test//1456/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.
